chart/values.yaml

```diff
@@ -181,20 +181,37 @@ controller:
 
     # Backoff settings for the controller manager. These settings control the backoff behavior
     # when the controller encounters an error while fetching secrets from the SecretSource.
+    # For example given the following settings:
+    #   initialInterval: 5s
+    #   maxInterval: 60s
+    #   randomizationFactor: 0.5
+    #   multiplier: 1.5
+    #
+    # The backoff retry sequence might be something like:
+    #  5.5s, 7.5s, 11.25s, 16.87s, 25.3125s, 37.96s, 56.95, 60.95s...
```

Comment:
BTW, the rendered .mdx for the website isn't going to preserve these line breaks and formatting, so we may want to do something like:

````yaml
    # For example given the following settings:
    # ```yaml
    #   initialInterval: 5s
    #   maxInterval: 60s
    #   randomizationFactor: 0.5
    #   multiplier: 1.5
    # ```
    #
    # The backoff retry sequence might be something like:
    #  5.5s, 7.5s, 11.25s, 16.87s, 25.3125s, 37.96s, 56.95, 60.95s...
````